### ISSUE TITLE
Composer 2.0 non-compliant vendor namespace

### ISSUE THREAD

**Issue by nickfls** [mastani/laravel-google-static-map#8]
Upon recent upgrade to composer stable channel 2, I started to get errors about missing classes;
```
Class 'mastani\GoogleStaticMap\GoogleStaticMap' not found
Class 'mastani\GoogleStaticMap\Format' not found
```

When downgraded to composer stable channel 1 (`composer self --1`) the errors went away.

Suggestion: update vendor prefix to TitleCase.
Do you want me to open a PR, @mastani ?

**Comment by mastani** [comment]
Thanks for reporting, Please open a PR if you can.

**Comment by mastani** [comment]
Closed in #7, #9, #10.
